packages/components/inputs/styles.js

```diff
@@ -81,7 +86,7 @@ const getInputStyles = (props, theme) => {
       `,
     ];
   }
-  if (props.isReadOnly) {
+  if (props.isReadOnly || props.readOnly) {
```

Comment:
what happened here?
why do we support `readOnly` as well?

Reply:
It's the same as: https://github.com/commercetools/ui-kit/pull/1539#discussion_r481970368

And due to: https://github.com/commercetools/ui-kit/pull/1539/files/b3715b6dca38a7ce8be8618aa15e8112d604c837#r481972538